### ISSUE TITLE
.unreel() failure with non-wheelable

### ISSUE THREAD

**Issue by danielocdh** [pisi/Reel#175]
Hello, first of all thanks for this wonderful plugin.

I'm using reel inside a slide, I need to to stop it when the slide is not visible, and restart it when the slide comes visible again.

If there is a way to pause the reel, then at another moment, reset it and resume the playback please let me know.

I can provide the page for this problem, but privately, is that possible?

I try to unreel like this:
`$reel_element.unreel();`
OR
`$('#container .reel-overlay').unreel();`
OR
`$('#container').unreel();`
OR
`$.reel.instances.unreel();`

for all of them I get this error on opera and chrome:

opera:

```
Error thrown at line 2, column 49306 in <anonymous function: acceptData>(e) in jquery.js:
    var t=e.nodeName&&st.noData[e.nodeName.toLowerCase()];
called from line 2, column 732 in i(e, n, r, i) in jquery.js:
    if(st.acceptData(e))
called from line 2, column 49128 in <anonymous function: _data>(e, t, n) in jquery.js:
    return i(e,t,n,!0)
called from line 3, column 5636 in <anonymous function: remove>(e, t, n, r, i) in jquery.js:
    var o,a,s,u,l,c,f,p,d,h,g,m=st.hasData(e)&&st._data(e);
called from line 3, column 6289 in <anonymous function: remove>(e, t, n, r, i) in jquery.js:
    st.event.remove(e,d+t[l],n,r,!0);
called from line 3, column 6289 in <anonymous function: remove>(e, t, n, r, i) in jquery.js:
    st.event.remove(e,d+t[l],n,r,!0);
called from line 3, column 6289 in <anonymous function: remove>(e, t, n, r, i) in jquery.js:
    st.event.remove(e,d+t[l],n,r,!0);
...
```

chrome:

```
Uncaught RangeError: Maximum call stack size exceeded
st.event.remove
st.event.remove
st.event.remove
st.event.remove
st.event.remove
st.event.remove
st.event.remove
st.event.remove
...
```

I'm using version Version: 1.2-devel Updated: 2013-02-21, because I need "scrollable" option, also using jquery 1.9.0

Thank you

**Comment by pisi** [comment]
Hi, please show me on **petr at vostrel dot cz**. Despite your issue being very interesting, be aware that you are working with a non-stable development code which is right now being prepared for release and still has some issues.

**Comment by pisi** [comment]
I just re-run the complete test suite on both Chrome on Opera (on both Windows and Mac) and despite everyone of these tests use `.unreel()` eventually, none of them resulted in loops described above. Dude, you gotta show me, really! ;)

**Comment by danielocdh** [comment]
mail with link to test site send, thank you

**Comment by pisi** [comment]
Nothing arrived yet.. (unless you were a viagra spam.. :D)

**Comment by danielocdh** [comment]
I sent the mail again from gmail and hotmail, this time copy pasted your email from your profile and changed the subject, just in case the previous mails didn't arrive, sorry for the spam if they did. Thank you.

**Comment by pisi** [comment]
Still nothing :/ Mighty strange! Try **vostrel at gmail com** and also please try sending me an e-mail without the link (to test if some link guard isn't acting upon us). Thanks.

**Comment by pisi** [comment]
Funnily enough the solution was to replace String nothing (`""`) with _real nothing_ (`null`) ;) to restore, how jQuery didn't bind prior to 1.9.x. A real bug after all! Was a tough one to spot [here](/pisi/Reel/blob/0600c230d7897fa58e56b4df59d8c1db225d8f8a/jquery.reel.js#L903).

Fix merged into the development branch, so @danielocdh, please grab [the latest patched code](/pisi/Reel/blob/development/jquery.reel.js) and confirm it is working in your application. Thanks!

**Comment by danielocdh** [comment]
Working perfectly on the example now, even that my example was a bit bad coded. Thanks a lot.

**Comment by pisi** [comment]
Awesome!

**Comment by pisi** [comment]
Merged into [master](/pisi/Reel/tree/master) and **included in today's [version 1.2.1](/pisi/Reel/tree/v1.2.1)** release.
